### PR TITLE
support space in workspace

### PR DESCRIPTION
**Pull request by jolorunyomi**
# Summary
Support space in WORKSPACE

### CODE REVIEW

File: ci/gpu/build.sh [hunk 1]
```diff
@@ -140,17 +140,17 @@ if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
...
             test_name=$(basename ${gt})
             echo "Running GoogleTest $test_name"
-            ${gt} --gtest_output=xml:${WORKSPACE}/test-results/
+            ${gt} --gtest_output=xml:"$WORKSPACE/test-results/"
         done
     fi
 else
     #Project Flash
-    export LIB_BUILD_DIR="$WORKSPACE/ci/artifacts/cudf/cpu/libcudf_work/cpp/build"
+    export LIB_BUILD_DIR="$WORKSPACE/ci/artifacts/cudf/cpu/libcudf"_work/cpp/build"
```
Comment (ajschmidt8) [inline ci/gpu/build.sh:153]:
```suggestion
    export LIB_BUILD_DIR="$WORKSPACE/ci/artifacts/cudf/cpu/libcudf_work/cpp/build"
```

File: ci/gpu/build.sh [hunk 2]
```diff
@@ -205,21 +205,21 @@ fi
...
+cd "$WORKSPACE/python/cudf"
 gpuci_logger "Python py.test for cuDF"
-py.test -n 6 --cache-clear --basetemp=${WORKSPACE}/cudf-cuda-tmp --junitxml=${WORKSPACE}/junit-cudf.xml -v --cov-config=.coveragerc --cov=cudf --cov-report=xml:${WORKSPACE}/python/cudf/cudf-coverage.xml --cov-report term
+py.test -n 6 --cache-clear --basetemp="$WORKSPACE/cudf-cuda-tmp" --junitxml="$WORKSPACE/junit-cudf.xml" -v --cov-config=.coveragerc --cov=cudf --cov-report=xml:"$WORKSPACE/python/cudf/cudf-coverage.xml" --cov-report term
 
-cd $WORKSPACE/python/dask_cudf
+cd "$WORKSPACE/python/dask"_cudf
 gpuci_logger "Python py.test for dask-cudf"
-py.test -n 6 --cache-clear --basetemp=${WORKSPACE}/dask-cudf-cuda-tmp --junitxml=${WORKSPACE}/junit-dask-cudf.xml -v --cov-config=.coveragerc --cov=dask_cudf --cov-report=xml:${WORKSPACE}/python/dask_cudf/dask-cudf-coverage.xml --cov-report term
+py.test -n 6 --cache-clear --basetemp="$WORKSPACE/dask-cudf-cuda-tmp" --junitxml="$WORKSPACE/junit-dask-cudf.xml" -v --cov-config=.coveragerc --cov=dask_cudf --cov-report=xml:"$WORKSPACE/python/dask"_cudf/dask-cudf-coverage.xml --cov-report term
```
Comment (ajschmidt8) [inline ci/gpu/build.sh:214]:
```suggestion
py.test -n 6 --cache-clear --basetemp="$WORKSPACE/dask-cudf-cuda-tmp" --junitxml="$WORKSPACE/junit-dask-cudf.xml" -v --cov-config=.coveragerc --cov=dask_cudf --cov-report=xml:"$WORKSPACE/python/dask_cudf/dask-cudf-coverage.xml" --cov-report term
```

File: ci/gpu/build.sh [hunk 3]
```diff
@@ -205,21 +205,21 @@ fi
...
 ################################################################################
 
-cd $WORKSPACE/python/cudf
+cd "$WORKSPACE/python/cudf"
 gpuci_logger "Python py.test for cuDF"
-py.test -n 6 --cache-clear --basetemp=${WORKSPACE}/cudf-cuda-tmp --junitxml=${WORKSPACE}/junit-cudf.xml -v --cov-config=.coveragerc --cov=cudf --cov-report=xml:${WORKSPACE}/python/cudf/cudf-coverage.xml --cov-report term
+py.test -n 6 --cache-clear --basetemp="$WORKSPACE/cudf-cuda-tmp" --junitxml="$WORKSPACE/junit-cudf.xml" -v --cov-config=.coveragerc --cov=cudf --cov-report=xml:"$WORKSPACE/python/cudf/cudf-coverage.xml" --cov-report term
 
-cd $WORKSPACE/python/dask_cudf
+cd "$WORKSPACE/python/dask"_cudf
```
Comment (ajschmidt8) [inline ci/gpu/build.sh:212]:
```suggestion
cd "$WORKSPACE/python/dask_cudf"
```

**Review by ajschmidt8 — CHANGES_REQUESTED** [review]
found a couple errors

**Comment by jolorunyomi** [comment]
> found a couple errors

Thanks for 👁️ @ajschmidt8

File: ci/cpu/build.sh [hunk 4]
```diff
@@ -64,7 +64,7 @@ if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
   CONDA_CHANNEL=""
 else
   CONDA_BUILD_ARGS="--dirty --no-remove-work-dir"
-  CONDA_CHANNEL="-c $WORKSPACE/ci/artifacts/cudf/cpu/.conda-bld/"
+  CONDA_CHANNEL="-c "$WORKSPACE/ci/artifacts/cudf/cpu/.conda-bld/"
```
Comment (ajschmidt8) [inline ci/cpu/build.sh:67]:
```suggestion
  CONDA_CHANNEL="-c $WORKSPACE/ci/artifacts/cudf/cpu/.conda-bld/"
```

I think this may be fine as it was.

**Review by ajschmidt8 — CHANGES_REQUESTED** [review]
found one more that might require review

**Review by ajschmidt8 — APPROVED** [review]

**Comment by harrism** [comment]
@jolorunyomi @ajschmidt8 should this be merged?

**Comment by jolorunyomi** [comment]
> @jolorunyomi @ajschmidt8 should this be merged?

Yes please this should be merged

**Review by GaryShen2008 — APPROVED** [review]

**Comment by harrism** [comment]
@gpucibot merge

**Pull request merged**